src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatWidget.ts

```diff
@@ -139,7 +140,16 @@ export class TerminalChatWidget extends Disposable {
 				}
 			},
 		);
-		this._register(this._inlineChatWidget.chatWidget.onDidChangeViewModel(() => this._saveViewState()));
+		this._register(autorun(r => {
```

Comment:
I'm seeing the old answer come back sometimes now.

Repro:

1. cmd+i
2. list files
3. cmd+enter
4. cmd+i, 🐛 list files Q&A exists

Reply:
I can reproduce that in insider's